Xamarin.Forms.Core/Shell/BaseShellItem.cs

```diff
@@ -185,6 +241,11 @@ static void OnIconChanged(BindableObject bindable, object oldValue, object newVa
 			shellItem.FlyoutIcon = (ImageSource)newValue;
 		}
 
+		internal void NotifyBadgePropertyChanged(string propertyName)
```

Comment:
BP supports INPC on their own. you can drop this

Reply:
Dropped.